pages/linux/pacstrap.md

```diff
@@ -0,0 +1,36 @@
+# pacstrap
+
+> Install packages using Pacman to the specified new root directory.
+> More information: <https://man.archlinux.org/man/pacstrap.8>.
+
+- Install the `base` package, Linux kernel and firmware for common hardware:
+
+`pacstrap {{path/to/new/root}} {{base}} {{linux}} {{linux-firmware}}`
+
+- Install packages without copy the host’s mirrorlist to the target:
+
+`pacstrap -M {{path/to/new/root}} {{packages}}`
+
+- Use an alternate configuration file for Pacman:
+
+`pacstrap -C {{path/to/pacman.conf}} {{path/to/new/root}} {{packages}}`
+
+- Install packages using the package cache on the host, rather than the target:
+
+`pacstrap -c {{path/to/new/root}} {{packages}}`
+
+- Install packages without copy the host's pacman keyring to the target:
+
+`pacstrap -G {{path/to/new/root}} {{packages}}`
+
+- Install packages in interactive mode (prompts user for confirmation):
```

Comment:
The reader _is_ the user :)
```suggestion
- Install packages in interactive mode (prompting for confirmation):
```

Reply:
I used `prompts` from https://github.com/tldr-pages/tldr/blob/master/pages/linux/cp.md page

Reply:
Hmm...well maybe we change it there and everywhere, what do you think?  Because though intuitive to write, from a reader's/user's perspective, it may look a little awkward.  @sbrl

Reply:
I'd just say `prompts for confirmation` to be honest. 🤷🏻

Reply:
I like that too.
```suggestion
- Install packages in interactive mode (prompts for confirmation):
```